tools/isobuild/resolver.ts

```diff
@@ -21,18 +21,25 @@ import {
   optimisticReadJsonOrNull,
 } from "../fs/optimistic";
 
-const nativeModulesMap: Record<string, string> = Object.create(null);
-const nativeNames = Object.keys((process as any).binding("natives"));
-
-// Node 0.10 does not include process as a built-in module, but later
-// versions of Node do, and we provide a stub for it on the client.
-nativeNames.push("process");
+// Note: the source code for the following function was copied from the 'builtin-modules' npm package (ver. 3.1.0)
+function getBuiltinModules() {
```

Comment:
According to my comments below, I don't see a use for this function anymore and we can just use `require('module').builtinModules`

Reply:
OK. The code has been changed to reflect this simplification.